docs/ContainerizedSetup.asciidoc

```diff
@@ -171,116 +168,128 @@ created for upgrade tests) and createhdds.sh is in `openqa_fedora_tools`
+    cd container/webui
+    docker-compose up
 
-Check whether the worker connected in the web UI's administration interface.
+To rebuild the images, add `--build`.
 
-To add more workers, increase the number that is used in hostname and
-container name, so to add worker 2 use:
+It is also possible to run it in the background by adding `-d`. To stop it
+again, run:
+
+    docker-compose down
+
+Further useful commands:
 
-    docker run -d -h openqa_worker_2 --name openqa_worker_2 --network webui_default --volumes-from openqa_data --privileged openqa_worker
+    docker-compose top                         # show spawned containers and their status
+    docker-compose logs                        # access logs
+    docker-compose exec db psql openqa openqa  # open psql shell
 
-==== Run a pool of workers automatically
+=== Generate and configure API credentials
+Go to https://localhost/api_keys and generate a key/secret and configure it in
+`container/webui/conf/client.conf` *and* `container/worker/conf/client.conf` in
+all sections.
+
+The web UI services need the credentials as well for internal API requests. So
+it is required to restart the web UI containers to apply the changes:
+
+    cd container/webui
+    docker-compose restart
+
+=== Run the worker container
+Configure the number of workers to start via the environment variable
+`OPENQA_WORKER_REPLICAS`. By default, one worker is started.
+
+To start a worker, just run:
+
+    cd container/worker
+    docker-compose up
+
+The same `docker-compose` commands as shown for the web UI can be used for
+further actions. The worker should also show up in the web UI's workers table.
+
+It is also possible to use Docker directly as shown by the script
+`container/worker/launch_workers_pool.sh` which allows spawning a bunch of
+workers with consecutive numbers for the `--instance` parameter:
 
-To launch a pool of workers one could use the script `./launch_workers_pool.sh`.
 It will launch the desired number of workers in individual containers using
-consecutive numbers for the `--instance` parameter.
+consecutive numbers for the `--instance` parameter:
 
-    ./launch_workers_pool.sh <number-of-workers>
+    cd container/worker
+    ./launch_workers_pool.sh --size=<number-of-workers>
 
 === Get tests, ISOs and create disks
-
 You have to put your tests under `data/tests` directory and ISOs under
 `data/factory/iso` directory. For testing openSUSE, follow
 https://github.com/os-autoinst/openQA/blob/master/docs/GettingStarted.asciidoc#testing-opensuse-or-fedora[this guide].
 
 == Running jobs
+After performing the "setup" tasks above - do not forget about tests and ISOs.
+
+Then you can use `openqa-cli` as usual with the containerized web UI.
+
+It is also possible to use `openqa-clone-job`, e.g. for the docker-compose setup
+one would call it like this:
+
+    script/openqa-clone-job \
```

Comment:
If openQA is installed then it should not be necessary to prefix with "script/", I suggest to just write `openqa-clone-job`

Reply:
Hm, in this case I am not sure it would even be possible to call the script successfully, think of the dependencies. IMHO would be more consistent to run that script from a container as well, similar to how you do it later with `docker exec`